seed/migrations/permitData.py

```diff
@@ -0,0 +1,102 @@
+
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.postgres.fields.jsonb
+import django_extensions.db.fields
+import autoslug.fields
+import django.utils.timezone
+import django.db.models.deletion
+from django.conf import settings
+
+class Migration(migrations.Migration):
+
+#From my understanding, the dependencies would be present if we are using another table/app. We are not doing so in these tables
+    dependencies = [
+    ]
+
+    operations = [
+      migrations.CreateModel(
+            name='seed_propertystate',
```

Comment:
seed_propertystate already exists in SEED. Did you run this migration to double check on your code if it actually works?